paper/paper.Rmd

```diff
@@ -23,15 +23,24 @@ knitr::opts_chunk$set(echo = TRUE)
 
 # Summary 
 
-Data are increasingly wider; in modern genomics, for example, high-resolution genetic data captures much more information than it did just a decade ago. While improved measurements contribute immensely to science, they also increase computational burden and complicate interpretability [@karczewski2018]. Data reduction techniques, such as principal components analysis and k-means clustering, are vital tools used to address these issues, particularly for noise and redundancy. However, these techniques may lead to problems in scalability, information loss, and interpretability [@malod-dogin-2018]. The Partition framework is an approach to data reduction that is flexible, scalable, and interpretable, developed to address information loss while maintaining speed[@Millstein2019].
+Data are increasingly feature-rich, including many variables for each observation; in modern genomics, for example, high-resolution genetic data captures much more information than it did just a decade ago. While improved measurements contribute immensely to science, they also increase computational burden and complicate interpretability [@karczewski2018]. Data reduction techniques, such as principal components analysis (PCA) and K-Means clustering, are vital tools used to address these issues, particularly for noise and redundancy. However, these techniques may lead to problems in scalability, information loss, and interpretability [@malod-dogin-2018]. The Partition framework is an approach to data reduction that is flexible, scalable, and interpretable, developed to address information loss while maintaining speed [@Millstein2019]. As opposed to other data reduction strategies, Partition only clusters data if the data reduction retains a certain amount of information. This framework is also agnostic to how clusters form; users can easily use other tools such as PCA and t-Distributed Stochastic Neighbor Embedding (t-SNE) to reduce data while constraining information loss.
 
-The R[@r-base] package partition is a fast and flexible data reduction tool that implements the Partition framework. partition enforces a minimum level of information, specified by the user, that reduced features must capture. Each feature begins as an independent cluster. Then, a bottom-up (agglomerative) approach grows clusters as much as possible, subject to the information loss constraint. Then, partition summarizes each feature in the subset into a single new feature. The reduced features are highly interpretable: original features map to one and only one feature in the reduced data set. The partition software is flexible, as well: how features are agglomerated, how information loss is measured, and the way data are reduced can all be customized. Additionally, partition has been thoroughly benchmarked and profiled, with key components written in C++ to improve performance.
```

Comment:
Suggested third sentence onward: 
Then, a bottom-up (agglomerative) approach grows clusters as much as possible, subject to the information loss constraint. Each partition subset of original features is summarized into a single new feature. The reduced features are highly interpretable because original features map to one and only one feature in the reduced data set. The partition software is flexible and customizable in the way features are agglomerated, information is measured, and data are reduced. Additionally, partition has been thoroughly benchmarked and profiled, with key components written in C++ to improve performance.